src/vmm/tests/integration_tests.rs

```diff
@@ -21,20 +20,6 @@ use vmm::vmm_config::instance_info::{InstanceInfo, VmState};
 use vmm::vmm_config::snapshot::{CreateSnapshotParams, SnapshotType};
 use vmm::{DumpCpuConfigError, EventManager, FcExitCode};
 
-#[test]
```

Comment:
All usage of `setup_serial_device` outside this test uses `Stdin` as the input. To support this test would require the significant change of adding a const generic to this function which would affect all the non-test code.

Reply:
I mentioned this before but removing test coverage because it needs significant change raises some concerns. If it is difficult to add these testcases now would you maybe like to create a github issue to add the test coverage that was removed in this PR?